### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by ha-manel**
In this branch I:

- Added linters config file
- Added a readme file
- Created HTML elements for header and headline sections (mobile version)
- Created CSS styles for the header and headline sections (mobile version)
- Added icons from fontawesome.com
- Added font style from fonts.google.com

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Manel Hammouche Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" id="logo">Manel</a>
+        <i class="fa-solid fa-bars fa-2x" id="menu_icon"></i>
+    </header>
```
Comment (mahbubul14) [inline index.html:17]: - [ ] The hamburger button on the right should have a `nav` tag. :writing_hand:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <div id="social_media_links">
+            <a href="https://twitter.com/ha_manel_" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-twitter fa-2x"></i></a>
+            <a href="https://www.linkedin.com/in/manel-hammouche/" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-linkedin fa-2x"></i></a>
+            <a href="https://github.com/ha-manel" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-github fa-2x"></i></a>
+            <a href="https://www.hackerrank.com/manelhammouche3" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-hackerrank fa-2x"></i></a>
+        </div>
```
Comment (mahbubul14) [inline index.html:35]: - [ ] Kindly use `ul` tag instead of `div` tag and `a` tag should be in `li` tag for each icon here. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @ha-manel ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Please add "Getting Started" section on your `README` file.

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first word will be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ha-manel** [comment]
@mahbubul14 Thank you for your review and all your suggestions, I'm working on them right now. However, concerning the "Getting Started" section in the readme file, I don't know how any of those titles apply to my small project (I have nothing to write for "prerequisites, setup, install, test...) if you have any suggestions I'd be very grateful to you!

**Comment by mahbubul14** [comment]
Hey @ha-manel, You can check [this link](https://github.com/mahbubul14/house_rent) to get an idea. Those parts you haven't worked on like "test". You don't need to add. You can just add a basic idea of how people can run the project.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @ha-manel   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Well implemented Figma design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**